src/changelog/unreleased.md

```diff
@@ -156,6 +156,7 @@ changelog entry.
     identify a finger in a multi-touch interaction. Replaces the old `Touch::id`.
   - In the same spirit rename `DeviceEvent::MouseMotion` to `PointerMotion`.
   - Remove `Force::Calibrated::altitude_angle`.
+- Update `smol_str` to version `0.3`
```

Comment:
The indentation is wrong.

Reply:
How so?
![image](https://github.com/user-attachments/assets/06bd4fb2-1e9e-48f9-9d8e-beb8f37d34d1)

Reply:
My mistake! I didn't check the context that it was in.